File: bank-service/src/main/java/rs/edu/raf/dto/PaymentQueueDTO.java
package rs.edu.raf.dto;

import java.math.BigDecimal;

public record PaymentQueueDTO(Long id,
                              String email,
                              Long senderAccountNumber,
                              Long receiverAccountNumber,
                              BigDecimal amount,
                              Long userId,
                              String type,
                              String paymentCode) {
}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/dto/PaymentQueueDTO.kt
package org.example.bankservicekotlin.dto

import java.math.BigDecimal

data class PaymentQueueDTO(
    val id: Long,
    val email: String,
    val senderAccountNumber: Long,
    val receiverAccountNumber: Long,
    val amount: BigDecimal,
    val userId: Long,
    val type: String,
    val paymentCode: String
)
